### ISSUE TITLE
Agent installation and service registration

### ISSUE THREAD

**Issue by ameir** [samdunne/chef-consul#1]
This cookbook seems pretty neat, and I'm about to give it a whirl in a few.  I noticed that it just interacts with the API; would you be open to a PR that installs the agent and allows for services to be registered?

There's another cookbook that does all that (https://github.com/johnbellone/consul-cookbook), but if you look at its dependencies, you'll go nuts (at least I did).  I'm just looking for a simple cookbook that gets the 95% nail hammered.

**Comment by samdunne** [comment]
Yeah absolutely. And that cookbook is dependency minefield alright.

It wasn't the original scope of this cookbook to install and configure consul. However it might make sense to have it do this.
